docs/content/production.html.md

```diff
@@ -453,6 +453,7 @@ To rotate an authentication key:
     flynn -a redis env set CONTROLLER_KEY=$NEW_KEY
     flynn -a mariadb env set CONTROLLER_KEY=$NEW_KEY
     flynn -a mongodb env set CONTROLLER_KEY=$NEW_KEY
+    flynn -a status env set AUTH_KEY=$NEW_KEY
```

Comment:
The status `AUTH_KEY` is intentionally not the same as the controller key, as it is used in monitoring systems that don't need full access to Flynn. This line should not be added to the documentation or tested below.